lib/expedia.js

```diff
@@ -86,8 +86,8 @@ module.exports = function (options, sendAsREST) {
                 return {
                     HotelListResponse: {
                         HotelList: {
-                            '@size': 0,
-                            '@activePropertyCount': 0,
+                            '@size': '0',
+                            '@activePropertyCount': '0',
                             HotelSummary: []
```

Comment:
Check the code please.

Reply:
this is what I was looking for as the answer `Expedia send back every number as a string. So that is why I named the PR to be consistent.`
You can mention the reason in the PR description, but this looked counter-intuitive which is why I asked.
IMO it does make sense to parse it to a number. Not sure why they send it as string. If you don't parse it FE will parse it to be usable and can result in a crash.

Reply:
yep, lets parse